src/components/forms/event-form.js

```diff
@@ -701,7 +701,14 @@ render() {
                     />
                 </div>
                 <div className="col-md-3">
-                    <label> {T.translate("edit_event.submitter")} </label>
+                    <label> {T.translate("edit_event.submitter")} </label> &nbsp;
+                        <i className='copy-button fa fa-copy'
```

Comment:
this is not the correct class should be 
https://fontawesome.com/v4/icon/clipboard